scenarios/cyrille37/sympa_ww-http-scan.yaml

```diff
@@ -0,0 +1,38 @@
+#
```

Comment:
The comments can be moved to a `.md` file as the same name as the `.yaml` which will be shown on hub.crowdsec.net